corehq/motech/management/commands/create_repeat_records.py

```diff
@@ -78,8 +76,9 @@ def doc_iterator(doc_ids):
                     except Exception:
                         logger.exception(f"Unable to fetch doc '{doc_id}'")
 
-        bulk_accessor = FormAccessorSQL.get_forms if options['doc_type'] == 'form' else CaseAccessorSQL.get_cases
-        single_accessor = FormAccessorSQL.get_form if options['doc_type'] == 'form' else CaseAccessorSQL.get_case
+        forms = XFormInstance.objects
+        bulk_accessor = forms.get_forms if options['doc_type'] == 'form' else CaseAccessorSQL.get_cases
```

Comment:
There isn't a `get_forms` method on the manager yet so this will need to keep using the accessor

Reply:
Oops, this slipped into the wrong PR. Great catch! 8923fe59caaf4dbed7687a3eec495c05ee4766e1